packages/bpk-component-datatable/src/BpkDataTable.module.scss

```diff
@@ -71,7 +71,7 @@ $bpk-border-size-datatable: $bpk-border-size-xl * 2;
     }
 
     &--selected {
-      background-color: $bpk-color-sky-gray-tint-06;
+      background-color: $bpk-canvas-contrast-day;
```

Comment:
Is selected to be contrast day? I see in the designs hover is contrast but selected is warning fill?

<img width="800" alt="Screenshot 2022-09-28 at 12 15 45" src="https://user-images.githubusercontent.com/8831547/192765548-ca5609ef-ca3b-4b58-8e3c-b06f90db6c56.png">

Reply:
ohh yes, you are right!!! good catch! will update it now